### ISSUE TITLE
Feature Request: Support for Batch API

### ISSUE THREAD

**Issue by RonaldFindling** [Asana/java-asana#95]
It would be amazing to have access to the Batch API as already requested here #75.
In our current usecases we update many tasks on a regular basis, this takes quite long without the Batch API. Are there plans to support this in the java-client?

**Comment by rossgrambo** [comment]
Yes and no. We're working on upgrading the tooling around how these are generated and the Batch API endpoint _should_ come along with that upgrade.

That being said, while Batch API is on our roadmap, it's currently not a priority. So if our tooling upgrades end up not supporting it (unlikely, but possible), then I'm not sure we would prioritize adding it.

It's good to have more examples of people requesting the feature, so i'll keep this issue open.

**Comment by OliverAbdulrahim** [comment]
Hey, following up on this.

It looks like the `BatchApis` resource was [generated here a couple of years ago](https://github.com/Asana/java-asana/blob/master/src/main/java/com/asana/resources/BatchApis.java) (after this feature request was submitted). I didn't see it exposed in `Client`, so I added it in a PR here: #156. 

I read #75 and if `BatchApis` isn't ready to be exposed to the public api, feel free to close the PR without merging.  (It is [mentioned in the public documentation.](https://developers.asana.com/docs/submit-parallel-requests))
